modules/control-utility/src/main/java/org/apache/ignite/internal/commandline/CommonArgParser.java

```diff
@@ -354,6 +361,11 @@ ConnectionAndSslParameters parseAndValidate(Iterator<String> rawArgIter) {
                         experimentalEnabled = true;
                         break;
 
+                    case CMD_SSL_FACTORY:
+                        sslFactoryCfg = argIter.nextArg("Expected SSL factory config path");
```

Comment:
What do you think about print warning or throw an exception when defined CMD_SSL_FACTORY and other cmdline SSL options (e.g. CMD_SSL_PROTOCOL, CMD_SSL_PROTOCOL etc)? Could this confuse the user?

Reply:
Done